packages/core/src/exposed-thing.ts

```diff
@@ -204,11 +208,12 @@ export default class ExposedThing extends ConsumedThing implements TD.Thing, WoT
         this.interaction.push(newProp);
 
         // FIXME does it makes sense to push the state to the ResourceListener?
-        let propState = new InteractionState();
-        propState.value = property.value;
-        propState.handlers = [];
-
-        this.interactionStates.set(newProp.name, propState);
+        let state = new PropertyState();
+        if(property.value != null) {
```

Comment:
Please follow code style conventions.
Visual Studio Code does it right, just "format document" before committing.